app/src/main/java/io/gnosis/safe/ui/settings/owner/OwnerEnterNameViewModel.kt

```diff
@@ -63,6 +63,27 @@ class OwnerEnterNameViewModel
             }
         }
     }
+
+    fun importLedgerOwner(address: Solidity.Address, name: String, derivationPathWithIndex: String) {
+        safeLaunch {
+            credentialsRepository.saveLedgerOwner(derivationPathWithIndex, address, name)
+            settingsHandler.showOwnerBanner = false
+            settingsHandler.showOwnerScreen = false
+            tracker.logLedgerKeyImported()
+            tracker.setNumKeysLedger(credentialsRepository.ownerCount(Owner.Type.LEDGER_NANO_X))
+            notificationRepository.registerSafes()
```

Comment:
this is not necesary at the moment. Maybe just leave a TODO, so that we can address ledger key registration later.